charter.md

```diff
@@ -0,0 +1,32 @@
+# Style Team
+
+## Aims: Evolving the Rust Style over time
+
+1. Selecting styling for new Rust constructs
+1. Evolving the existing style over the course of Rust editions (without breaking backwards compatibility)
+1. Defining mechanisms to evolve the Rust style while taking backwards compatibility into account, such as via Rust editions or similar mechanisms
```

Comment:
```suggestion
1. Selecting styling for new Rust constructs
2. Evolving the existing style over the course of Rust editions (without breaking backwards compatibility)
3. Defining mechanisms to evolve the Rust style while taking backwards compatibility into account, such as via Rust editions or similar mechanisms
```

I know markdown auto-numbers, but people will often read the text versions, so let's make them more readable.